test/integration_sops_test.go

```diff
@@ -0,0 +1,99 @@
+package test_test
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"testing"
+
+	"github.com/gruntwork-io/terragrunt/util"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	testFixtureSops       = "fixtures/sops"
+	testFixtureSopsErrors = "fixtures/sops-errors"
+)
+
+func TestSopsDecryptedCorrectly(t *testing.T) {
+	t.Parallel()
+
+	cleanupTerraformFolder(t, testFixtureSops)
+	tmpEnvPath := copyEnvironment(t, testFixtureSops)
+	rootPath := util.JoinPath(tmpEnvPath, testFixtureSops)
+
+	runTerragrunt(t, "terragrunt apply -auto-approve --terragrunt-non-interactive --terragrunt-working-dir "+rootPath)
+
+	stdout := bytes.Buffer{}
+	stderr := bytes.Buffer{}
+
+	err := runTerragruntCommand(t, "terragrunt output -no-color -json --terragrunt-non-interactive --terragrunt-working-dir "+rootPath, &stdout, &stderr)
```

Comment:
It can be complicated to debug since the output is not printed, can be used `runTerragruntCommandWithOutput()` which will return output and print